modules/nodes-scheduler-node-selectors-project.adoc

```diff
@@ -5,59 +5,113 @@
 [id="nodes-scheduler-node-selectors-project_{context}"]
 = Creating project-wide node selectors  
 
-You can use node selectors on a project together with labels on nodes to constrain all pods created in a namespace to the labeled nodes.
+You can use node selectors in a project together with labels on nodes to constrain all pods created in that project to the labeled nodes.
 
-With project node selectors, when you create a pod in the namespace, {product-title} adds the appropriate `<key>:<value>` and schedules
-the pod on nodes with matching labels. 
+When you create a pod in this project, {product-title} adds the node selectors to the pods in the project and schedules the pods on a node with matching labels in the project. If there is a cluster-wide default node selector, a project node selector takes preference.
 
-You can add labels to a node or MachineConfig, but the labels will not persist if the node or machine goes down. 
-Adding the label to the MachineSet ensures that new nodes or machines will have the label.
+You add labels to a project by editing the `Namespace` object. You add labels to a node by editing the Node object, a MachineSet, or a MachineConfig. Adding the label to the MachineSet ensures that if the node or machine goes down, new nodes have the label. Labels added to a node or MachineConfig do not persist if the node or machine goes down.
```

Comment:
```suggestion
You add labels to a project by editing the `Namespace` object to add the `openshift.io/node-selector` parameter, which contains the label definitions. You add labels to a node by editing the Node object, a MachineSet, or a MachineConfig. Adding the label to the MachineSet ensures that if the node or machine goes down, new nodes have the label. Labels added to a node or MachineConfig do not persist if the node or machine goes down.
```
Or change the first step to talk about adding the labels instead of the `openshift.io/node-selector` object. Or both.